addons/contexts/src/manager/components/ToolBarControl.tsx

```diff
@@ -47,5 +47,5 @@ export const ToolBarControl: ToolBarControl = ({
     },
   };
 
-  return icon && list.length && !options.disable ? <ToolBarMenu icon={icon} {...props} /> : null;
+  return list.length && !options.disable ? <ToolBarMenu icon={icon} {...props} /> : null;
```

Comment:
An extra check could be `Array.isArray(list) && list.length`

Reply:
👍 Added the extra check